### PR TITLE
treewide: remove BUILD_PARALLEL and PKG_INSTALL from CMake packages

### PR DESCRIPTION
**Pull request by neheb**
Not needed as they are default.

### CODE REVIEW

**Comment by neheb** [comment]
uci parallel builds were disabled in 2012 with https://github.com/openwrt/openwrt/commit/fc760260dde . This doesn't seem to be needed anymore but maybe @nbd168 has some insight.

**Comment by aparcar** [comment]
Please rebase.

**Comment by neheb** [comment]
Rebased.

File: package/system/uci/Makefile [hunk 1]
```diff
@@ -22,7 +22,6 @@ PKG_LICENSE_FILES:=
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 
-PKG_BUILD_PARALLEL:=0
```
Comment (aparcar) [inline package/system/uci/Makefile:25]: I'd just leave this line for now and move on with the rest.

Comment (neheb) [reply package/system/uci/Makefile:25]: It's not needed as it's being built with Ninja now. I've tested multiple clean compiles and it has not failed even once.

**Comment by aparcar** [comment]
Thanks for the cleanup.

**Pull request closed without merging**